src/components/AssetEventsPanel/interfaces.ts

```diff
@@ -65,12 +66,27 @@ export interface AssetEventsPanelThemeProps {
   theme?: AnyIfEmpty<{}>;
 }
 
+export interface EventAddonsProp extends CogniteEvent {
+  typeAndSubtype: React.ReactNode;
+  start: string;
+  end: string;
+}
+
+export type MetadataRenderer = (event: EventAddonsProp) => React.ReactNode;
+
+export interface RenderMetadataProps {
+  /** Function which receives event and returns rendered metadata */
+  renderEventMetadata?: MetadataRenderer;
```

Comment:
I believe we should specify which metadata
so something like `EventMetadataRenderer` and `EventRenderMetadataProps` should be more descriptive

Reply:
for the one above as well

Reply:
And fixed!